src/Client/ClientBase.cpp

```diff
@@ -1349,15 +1349,14 @@ void ClientBase::runInteractive()
     /// Initialize DateLUT here to avoid counting time spent here as query execution time.
     const auto local_tz = DateLUT::instance().getTimeZone();
 
-    std::optional<Suggest> suggest;
-    suggest.emplace();
+    Suggest suggest(suggest_interval_ms);
     if (load_suggestions)
     {
         /// Load suggestion data from the server.
         if (global_context->getApplicationType() == Context::ApplicationType::CLIENT)
-            suggest->load<Connection>(global_context, connection_parameters, config().getInt("suggestion_limit"));
+            suggest.schedule<Connection>(global_context, connection_parameters, config().getInt("suggestion_limit"));
```

Comment:
btw for what type of suggestions this is useful to update periodically?

Reply:
Table and databases names.